QtAwesome/iconic_font.py

```diff
@@ -10,12 +10,20 @@
 _default_options = {
     'color': QColor(50, 50, 50),
     'scale_factor': 0.9,
+    'rotation': False,      # Valid options are: 'spin', or 'pulse'
+    'parent': False,        # Ref to instance of widget holding the icon
+    'spin_interval': 10,    # miliseconds
+    'spin_step': 1,         # degrees
+    'pulse_interval': 300,  # miliseconds
+    'pulse_step': 45,       # degrees
```

Comment:
Regarding the aspect, I think that it is because ttf embed an already rendered bitmap for certain sizes. If you want to have a nice rendering for other sizes, we should use crispEdges styling. I did not do it because I could not see the problem occur.